run_gemini.py

```diff
@@ -0,0 +1,64 @@
+import google.generativeai as genai
+import os
+import subprocess
+import logging
+from dotenv import load_dotenv
+
+
+def get_review(api_key, commit_branch, main_branch, logger):
+    subprocess.run(
+        f"git fetch origin {main_branch} && git fetch origin {commit_branch} && git checkout {commit_branch}",
+        shell=True,
+        check=True,
+    )
```

Comment:
The code assumes `git diff` output always contains text. It might be possible that the `git diff` output is empty, leading to an unexpected error in the script.  Handling the case where `git diff` returns an empty string is important.